internal/plans/objchange/plan_valid_test.go

```diff
@@ -1256,3 +1486,85 @@ func TestAssertPlanValid(t *testing.T) {
 		})
 	}
 }
+
+func TestAssertPlanValidTEST(t *testing.T) {
```

Comment:
What does the "TEST" at the end of this function name represent? :thinking:

Reply:
Oops, that was my temporary test harness for trying out values. I'll delete that before merging!